packages/dev/loaders/src/glTF/2.0/Extensions/KHR_interactivity/declarationMapper.ts

```diff
@@ -0,0 +1,1689 @@
+                loopBody: { name: "executionFlow" },
+            },
+        },
+    },
+    "flow/doN": {
+        blocks: [FlowGraphBlockNames.DoN],
+        configuration: {},
+        inputs: {
+            values: {
+                n: { name: "maxExecutions", gltfType: "number" },
+            },
+        },
+        outputs: {
+            values: {
+                currentCount: { name: "executionCount" },
+            },
+        },
+    },
+    "flow/multiGate": {
+        blocks: [FlowGraphBlockNames.MultiGate],
+        configuration: {
+            isRandom: { name: "isRandom", gltfType: "boolean", inOptions: true, defaultValue: false },
+            isLoop: { name: "isLoop", gltfType: "boolean", inOptions: true, defaultValue: false },
+        },
+        extraProcessor(gltfBlock, declaration, _mapping, _arrays, serializedObjects) {
+            if (declaration.op !== "flow/multiGate" || !gltfBlock.flows || Object.keys(gltfBlock.flows).length === 0) {
+                throw new Error("MultiGate should have a single configuration object, the number of output flows");
+            }
+            const serializedObject = serializedObjects[0];
+            serializedObject.config = serializedObject.config || {};
+            serializedObject.config.outputSignalCount = Object.keys(gltfBlock.flows).length;
+            serializedObject.signalOutputs.forEach((output, index) => {
+                output.name = "out_" + index;
+            });
+            return serializedObjects;
+        },
+    },
+    "flow/waitAll": {
+        blocks: [FlowGraphBlockNames.WaitAll],
+        configuration: {
+            inputFlows: { name: "inputSignalCount", gltfType: "number", inOptions: true, defaultValue: 0 },
+        },
+        inputs: {
+            flows: {
+                "[segment]": { name: "in_$1" },
+            },
+        },
+        validation(gltfBlock) {
+            // check that the configuration value is an integer
+            if (typeof gltfBlock.configuration?.inputFlows?.value[0] !== "number") {
+                gltfBlock.configuration = gltfBlock.configuration || {
+                    inputFlows: { value: [0] },
+                };
+                gltfBlock.configuration.inputFlows.value = [0];
+            }
+            return true;
+        },
+        // extraProcessor(_gltfBlock, _declaration, _mapping, _arrays, serializedObjects) {
+        //     // process the input flows and add them to the inFlow array
+        //     // take all input flows and convert their names correctly to "in_$1"
+        //     const serializedObject = serializedObjects[0];
+        //     serializedObject.signalInputs.forEach((input) => {
+        //         input.name = "in_" + input.name;
+        //     });
+        //     return serializedObjects;
+        // },
+    },
+    "flow/throttle": {
+        blocks: [FlowGraphBlockNames.Throttle],
+        outputs: {
+            flows: {
+                // out: { name: "out" },
```

Comment:
Why commented out?

Reply:
this is mapping for readability, mapping to the same name is done automatically. I'll remove it.